src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -129,6 +129,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\Text\Base64.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\Text\Base64Encoder.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\Text\Base64Decoder.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Buffers\Text\Base64Url\Base64UrlDecoder.cs" />
```

Comment:
Should this live in System.Private.Uri with other Url stuff?

Reply:
One of the reasons of adding here is Base64Url shares existing Base64 implementation

Reply:
Can it reuse the existing Base64 implementation via the Base64 public APIs? If it is not possible, should we look at improving Base64 public APIs to make it actually usable?

Reply:
It's not really about Uri. Base64url is basically just Base 64 with a slightly different alphabet, and is used by things like JWT that aren't specific to Uri. I see this as actually improving the Base64 APIs, they're just grouped according to alphabet.